syntax = "proto2";

message c2g_player_handshake_request {
	required string key = 1;
}